packages/@aws-cdk/aws-s3-deployment/test/test.bucket-deployment.ts

```diff
@@ -255,7 +255,7 @@ export = {
         'storage-class': 'INTELLIGENT_TIERING',
         'server-side-encryption': 'AES256',
         'cache-control': 'public, max-age=3600',
-        'expires': new Date(Date.now() + 12 * 60 * 60 * 1000).toUTCString()
+        'expires': new Date(cdk.Duration.now + 12 * 60 * 60 * 1000).toUTCString()
```

Comment:
It feels a little repetitive, but simpler to just reuse `Expires.after(...).value` here.

(And in fact required if you're going to move `Date.now` into a private variable).